bolt/include/bolt/Core/DebugData.h

```diff
@@ -450,10 +456,16 @@ class DebugStrOffsetsWriter {
     return std::move(StrOffsetsBuffer);
   }
 
-private:
   /// Initializes Buffer and Stream.
   void initialize(DWARFUnit &Unit);
 
+  /// Clear data.
```

Comment:
This could be moved out of this diff into a small NFC change.

Reply:
I don't think it simplifies the review, and context for change is within this PR anyway.